android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -294,7 +272,7 @@ public void getLastPushPayload(Promise promise) {
 
     @ReactMethod
     public void disableDeviceForCurrentUser() {
-        IterableLogger.v(TAG, "Disable Device");
+        IterableLogger.v(TAG, "disableDevice");
```

Comment:
Im just checking to make sure these calls to IterableLogger follow the same convention.